### PR TITLE
feat(perception_launch): replace perception_launch with tier4_perception_launch

### PR DESCRIPTION
**Pull request by kminoda**
Signed-off-by: kminoda <koji.minoda@tier4.jp>

## PR Type

- Improvement

## Related Links
- Localization: https://github.com/tier4/autoware_launch/pull/651
- Perception: https://github.com/tier4/autoware_launch/pull/681

## Description
Replace perception_launch with tier4_perception_launch

The parameter configuration in autoware.universe and autoware_launch has been modified as described in [this issue](https://github.com/autowarefoundation/autoware_launch/issues/66).
With this modification, we are now able to pass parameter paths to tier4_xx_launch from the outside of the package, i.e. autoware_launch packages. Thus, I would like to remove xx_launch from tier4/autoware_launch and replace it with tier4_xx_launch in autoware.universe. 

With this PR, we can reduce the maintenance cost of xx_launch.

(Before this PR: we need to modify both tier4_xx_launch and xx_launch. After this PR: we only need to modify tier4_xx_launch) 

## Review Procedure
Please launch lsim and check that Autoware works the same as before.

## Remarks
Please refer to this [TIER IV internal link](https://tier4.atlassian.net/wiki/spaces/~727523079/pages/2673279578/tier4+autoware+launch+xx+launch+-+tier4+xx+launch), too.

Note to each product:
#### Most of the products
Set `use_traffic_light_recognition` to false in `tier4_perception_component.launch.xml` when disabling traffic light recognition.

#### XX1
The XX1 uses its original models. In order to provide the original model weights, you can set [these three parameters](https://github.com/autowarefoundation/autoware.universe/blob/316f5e9be575a804033554bfbe33c8a030caf4dc/launch/tier4_perception_launch/launch/perception.launch.xml#L12-L14) from `tier4_perception_component.launch.xml`.

#### S1
Please consider adding a new settings (or maybe a new launch file) for euclidean clustering-based detection configuration. Also related to: https://github.com/autowarefoundation/autoware.universe/pull/842

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [ ] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Review by yukkysaito — DISMISSED** [review]
LGTM
pls get approval by @tkimura4

File: autoware_launch/launch/autoware.launch.xml [hunk 1]
```diff
@@ -100,8 +100,9 @@
 
   <!-- Perception -->
   <group if="$(var launch_perception)">
-    <include file="$(find-pkg-share perception_launch)/launch/perception.launch.xml">
-      <arg name="mode" value="$(var perception_mode)" />
+    <include file="$(find-pkg-share autoware_launch)/launch/components/tier4_perception_component.launch.xml">
+      <arg name="mode" value="lidar"/>
```
Comment (tkimura4) [inline autoware_launch/launch/autoware.launch.xml]: Is it intentional to set the mode to lidar instead of $(var perception_mode)?

**Review by tkimura4 — DISMISSED** [review]
Looks ok except for the one comment above

File: autoware_launch/launch/autoware.launch.xml (hunk 1 above)
Comment (kminoda) [reply autoware_launch/launch/autoware.launch.xml]: Thank you, that is a mistake. Fixed here: https://github.com/tier4/autoware_launch/pull/681/commits/bf9a1ae23b6d4a743d7dfbd755ca2e28591cd9d0

**Comment by kminoda** [comment]
@tkimura4 @yukkysaito Sorry, would you approve this again? :bow:

**Review by yukkysaito — DISMISSED** [review]

**Comment by kminoda** [comment]
@tkimura4 @yukkysaito Sorry, would you approve this again? :bow:

**Review by tkimura4 — APPROVED** [review]

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**